queries/highlights.scm

```diff
@@ -53,13 +50,20 @@
   "while"
   "repeat"
   "for"
+  (next)
+  (break)
 ] @keyword
 
 [
-  (inf) (nan) (dots) (true) (false) (na)
+  (true)
+  (false)
+  (null)
+  (inf)
+  (nan)
+  (na)
+  (dots)
+  (dot_dot_i)
```

Comment:
In this file:
- `(return)` removed
- `(next)` and `(break)` moved to existing `@keyword` list
- `(null)` moved from `@keyword` to `@constant.builtin`
- `(dots)` and `(dot_dot_i)` added to `@constant.builtin`

My rationale for `@constant.builtin` is that those resolve to actual values, unlike `@keyword`